### PR TITLE
Desktop equip - ignore right clicks

### PR DESCRIPTION
**Pull request by dback2**
Test Plan:
https://highfidelity.testrail.net/index.php?/cases/view/15103
Also verify you can inspect equippable entities from the marketplace.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9273/app_13033_PR_78436ebcbf1cac571f470215e6d096d76e7a7215.apk

File: scripts/system/controllers/controllerModules/equipEntity.js [hunk 1]
```diff
@@ -796,7 +796,7 @@ EquipHotspotBuddy.prototype.update = function(deltaTime, timestamp, controllerDa
     };
     
     var onMousePress = function(event) {
-        if (isInEditMode()) { // don't consider any mouse clicks on the entity while in edit
+        if (isInEditMode() || event.isRightButton) { // don't consider any mouse clicks on the entity while in edit
```
Comment (sethalves) [inline scripts/system/controllers/controllerModules/equipEntity.js]: maybe this should be || !event.isPrimaryButton

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9273/HighFidelity-Beta-PR13033-78436ebcbf1cac571f470215e6d096d76e7a7215.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9273/HighFidelity-Beta-Interface-PR13033-78436ebcbf1cac571f470215e6d096d76e7a7215.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9273/HighFidelity-Beta-PR13033-78436ebcbf1cac571f470215e6d096d76e7a7215.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9273/HighFidelity-Beta-Interface-PR13033-78436ebcbf1cac571f470215e6d096d76e7a7215.dmg

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by conklin94122** [comment]
@dback2 Make same changes to https://github.com/highfidelity/hifi/pull/13036, yeah?

**Comment by dback2** [comment]
@conklin94122 yes changes coming now
@qlimnavarro hold off on testing

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9283/app_13033_PR_8d49d1c1181845d253a281116bd81fd13eabce0b.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9283/HighFidelity-Beta-PR13033-8d49d1c1181845d253a281116bd81fd13eabce0b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9283/HighFidelity-Beta-Interface-PR13033-8d49d1c1181845d253a281116bd81fd13eabce0b.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9283/HighFidelity-Beta-PR13033-8d49d1c1181845d253a281116bd81fd13eabce0b.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9283/HighFidelity-Beta-Interface-PR13033-8d49d1c1181845d253a281116bd81fd13eabce0b.exe

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Right click does not pick up the item and can access the certificate icon with ease. Certs still work as expected.

**Pull request merged**